modules/using-cluster-compare-must-gather.adoc

```diff
@@ -82,3 +82,18 @@ No patched CRs <10>
 <8> The CRs that did not match to a corresponding template in the reference configuration.
 <9> The metadata hash identifies the reference configuration.
 <10> The list of patched CRs.
+
+[NOTE]
+====
+You can also get the output in `junit` format by adding `-o junit` to the command. For example:
```

Comment:
```suggestion
You can also get the output in a `junit` format by adding `-o junit` to the command. For example:
```
https://www.ibm.com/docs/en/ibm-style?topic=grammar-articles

Please apply for all of the other similar instances in this PR.